src/utils/constant/week.ts

```diff
@@ -0,0 +1,3 @@
+const WEEK: string[] = ['월', '화', '수', '목', '금', '토', '일'];
```

Comment:
타입스크립트를 조금 더 자세히 적용한다면 드릴만한 팁은,

1.`string[]` 로 타입을 정의하지 않았어도 알아서 추론되었을거라 불필요합니다.
2. 정의한다면 `WEEK[i]` 는 `string`이 아닌 `'월' | '화' | '수'...` 로 추론될 수 있도록 `as const`를 붙여주면 더욱 변수명에 맞는 타입을 가질 수 있어요
```suggestion
const WEEK = ['월', '화', '수', '목', '금', '토', '일'] as const;
```

Reply:
타입스크립트를 사용하면서 조언해주신 부분에 대해서는 생각해보지 못한 것 같습니다! 꿀팁 감사합니다!!
해당 부분 수정했습니다. bf6b181fa41525c218f1060291b8d2353deea853